content/intro-to-storybook/react/en/using-addons.md

```diff
@@ -158,7 +158,3 @@ If we are using [visual regression testing](/react/en/test/), we will also be in
 ### Merge Changes
 
 Don't forget to merge your changes with git!
-
```

Comment:
Put all of this content in an `.aside`. Something like "Bonus chapter: Creating your own addon... link to the chapter"

Reply:
I don't think it should be removed altogether, it's useful to link to the bonus chapter here